servicetalk-grpc-api/src/main/java/io/servicetalk/grpc/api/GrpcStatus.java

```diff
@@ -162,9 +188,12 @@ public GrpcStatusCode code() {
      * Returns the cause, can be null.
      *
      * @return the cause, can be null.
+     * @deprecated Use {@link GrpcStatusException#fromThrowable(Throwable)} to create a {@link GrpcStatusException}
+     * then {@link GrpcStatusException#getCause()}.
      */
     @Nullable
-    public Throwable cause() {
+    @Deprecated
+    public Throwable cause() { // FIXME: 0.43 - remove deprecated method
```

Comment:
the relationship between `GrpcStatus` and `GrpcStatusException` has always been awkward. This change will move us toward `GrpcStatus` just being a status (not containing an exception) and using  `GrpcStatusException` when an exception is in play

Reply:
moved to https://github.com/apple/servicetalk/pull/2422